packages/react-topology/src/behavior/useDndDrag.tsx

```diff
@@ -303,7 +303,7 @@ export const useDndDrag = <
 };
 
 export interface WithDndDragProps {
-  dndDragRef: ConnectDragSource;
+  dndDragRef?: ConnectDragSource;
```

Comment:
The new `DefaultNode` is trying to be one-stop shopping. It can be used for nodes that are or are not draggable. Apps will have issues if we make these props mandatory.